src/components/AlertList.vue

```diff
@@ -6,9 +6,9 @@
       :items="alerts"
       :pagination.sync="pagination"
       :rows-per-page-items="pagination.rowsPerPageItems"
+      :total-items="pagination.totalItems"
       class="alert-table"
       :class="[ displayDensity ]"
-      :search="search"
```

Comment:
Why did you delete "search"?

Reply:
Remove "search" makes pagination work, maybe it's a bug for vuetify data-table.
https://github.com/vuetifyjs/vuetify/issues/5419